conf/modules.config

```diff
@@ -200,6 +210,56 @@ process {
     }
 }
 
+//
+// Sentieon alignment options
+//
+
+process{
+    withName: '.*ALIGN_SENTIEON:.*' {
+        ext.when = params.aligner.equals("sentieon")
+    }
+
+    withName: '.*ALIGN:ALIGN_SENTIEON:SENTIEON_BWAMEM' {
+        ext.args = { "-M -K 10000000 -R ${meta.read_group} " }
```

Comment:
Maybe let the users determine if they wanna set a specific chunk size at all?  
{ ${params.chunk_size} ? "-M -K ${params.chunk_size} -R ..." : "-M -R ..." }

Reply:
My reasoning for not doing that here is that one can change a lot of parameters in bwa (and other tools used in raredisease), and creating a specific parameter in the pipeline for those could get messy and confusing. 

I think if a user wants to change these defaults they should do so using a config file like [here](https://github.com/nf-core/configs/blob/9da85209f234e10d8e84b11d9bd7ee39a2a2fc4f/conf/pipeline/raredisease/hasta.config).